app/services/import_reaction_service.rb

```diff
@@ -0,0 +1,23 @@
+class ImportReactionService < ApplicationService
+  def initialize(post, reactions)
+   @post = post
+   @reactions = reactions
+  end
+
+  def call
+    reactions.each do |reaction|
```

Comment:
same for reactions, also is the `fid` attribute needed at all for that model?

Reply:
yeah, i though about that, but then though that it might be useful as single unique identifier , instead of using user_id & post_id, but yeah we could get rid of it